docs/dev/release.md

```diff
@@ -124,6 +124,93 @@ At this point, we should have deployed the release to Sonatype (and eventually t
 
 In addition, the project may be released on GitHub. This provides a way to add release notes for a particular project. A GitHub release is done manually by uploading artifacts through the UI. The release version and tag should be the same used for Maven Central. Release notes can be automatically generated from commit messages and then customized.
 
+----
+
+### `main`
+
+* Update POM version to release version
+* Push update to GH
+* Maven release
+* Push release tag and update to new snapshot version to GH
+* Maven deploy new snapshot
+
+### `pass-core`
+
+* Update POM version to release
+* Push update to GH
+* Maven release (include submodules)
+* Push generated release Docker image to GHCR
+* Push release tag to GH
+* Update top level and submodule POMs, allowing SNAPSHOTS
+* Push version updates on main to GH
+* Maven deploy new snapshot
+* Push snapshot Docker image to GHCR
+
+This follows the same general release procedure as `main` with a few extra steps to account for submodules and Docker images
+
+### `pass-support`
+
+See procedure for `pass-core`, but without Docker images
+
+``` sh
```

Comment:
Thoughts on moving this into a `RELEASE=0.2.0 NEXT=0.3.0-SNAPSHOT ./tools/release/xxxx` script to make the release a _one-liner_?

Reply:
Sure, I'd be for moving this over there. I had initially only wanted feedback about the completeness and correctness of the procedure in order to then translate it into a GH workflow, but I can see having a more full bash script like this available in `./tools` could be helpful as a fall back to the GH automation, especially if a GH automation isn't complete before the next release.

If we want this to serve as a one-liner for release it'll need some tweaks, but I think it's relatively close in its current state